Add dashboard for Rekor v2

cmurphy opened the pull request:
## Add dashboard for Rekor v2

Add a dashboard with widgets for:

- the total tree size
- rekor_v2_http_api_latency (the only latency observer defined in
  rekor-tiles)
- Tessera appender duration histogram
- the two SLOs (availabiltiy of gRPC and HTTP)
- the two error metrics defined in rekor-tiles, gRPC panics and
  inclusion proof errors

## Use lower-case SLO filter

promhttp seems to use a lower-case method name in its labels[1] so the
error rate was not being collected when trying to filter using the
upper-case version.

---
Fixes https://github.com/sigstore/rekor-tiles/issues/361

Open to suggestions on other dashboards we should include.

<!--
Thanks for opening a pull request! Please do not just delete this text.  The three fields below are mandatory.

Please remember to:
- This PR requires an issue. If it is a new feature, the issue should proceed the PR and will have allowed sufficient time for discussions to take place. Please use
issue tags such as "Closes #XYZ" or "Resolves sigstore/repo-name#XYZ".
  [documentation](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)
- ensure your commits are signed-off, as sigstore uses the [DCO](https://en.wikipedia.org/wiki/Developer_Certificate_of_Origin) using `git commit -s`, or `git commit -s --amend` if you want to amend already existing commits
- lastly, ensure there are no merge commits!

Thank you :)
-->

#### Summary
<!--
 Explain the **motivation** for making this change. What existing problem does the pull request solve? How can reviewers test this PR?
-->

#### Release Note
<!--
Add a release note for each of the following conditions:

* Config changes (additions, deletions, updates)
* API additions—new endpoint, new response fields, or newly accepted request parameters
* Database changes (any)
* Websocket additions or changes
* Anything noteworthy to an administrator running private sigstore instances (err on the side of over-communicating)
* New features and improvements, including behavioural changes, UI changes and CLI changes
* Bug fixes and fixes of previous known issues
* Deprecation warnings, breaking changes, or compatibility notes

If no release notes are required write NONE. Use past-tense.

-->

#### Documentation
<!--

Does this change require an update to documentation? How will users implement your new feature?

Please reference a PR within https://docs.sigstore.dev

-->

cmurphy commented:
I don't want to modify the JSON I copied from creating the dashboard, if we ever need to update it it will be easier to copy and update the same output.

haydentherapper commented:
> I don't want to modify the JSON I copied from creating the dashboard, if we ever need to update it it will be easier to copy and update the same output.

Unfortunately we have to remove the empty objects, see https://github.com/sigstore/terraform-modules/pull/82 and the issue linked in the PR for more context. Otherwise `tf plan` always shows a diff.

haydentherapper commented on gcp/modules/monitoring/infra/rekor_v2.json:
```diff
@@ -0,0 +1,229 @@
+                    "aggregation": {
+                      "perSeriesAligner": "ALIGN_MEAN"
+                    },
+                    "filter": "select_slo_health(\"projects/253269845889/services/log2025-alpha1-rekorv2/serviceLevelObjectives/grpc-server-availability-api-v2-log-entries-post\")"
```
This is specifically for one log shard - could we have it dynamically select the latest shard?

haydentherapper reviewed (commented):
Could we have a chart for both rekor v1 and rekor v2 traffic, to ideally show the respective decrease and increase in traffic over time?

cmurphy commented:
I moved this dashboard to the rekorv2 submodule so that it could refer to the SLO IDs in the dashboard template. I added another dashboard widget to show request traffic, and put up another PR at https://github.com/sigstore/terraform-modules/pull/101 (already merged) so that Rekor v1 could have its own traffic dashboard widget.
